lib/traject/macros/xml.rb

```diff
@@ -17,6 +17,10 @@ def extract_xml(xpath, namespaces, options = {})
       end
     end
 
+    def extract_field(xml, xpath, namespaces)
```

Comment:
is it expanded by the fgdc + mods xpath methods? i can check, but that's my understanding